aibolit/config.py

```diff
@@ -174,6 +174,8 @@ def get_patterns_config():
             },
             "patterns_exclude": [
                 "P27",  # empty implementation
+                "P20_5", "P20_7", "P20_11",  # wasn't refactored yet
+                "P28", "P9",  # patterns based on text cannot accept arbitrary AST
             ],
-            "metrics_exclude": []
+            "metrics_exclude": ["M1", "M3_1", "M3_2", "M3_3", "M3_4", "M5", "M6", "M7"]
```

Comment:
why we exclude metrics?

Reply:
They have been excluded in my PR. I did it, because they are not refactored. So we only have NCSS and Cognitive.